scripts/core/editor3/styles.scss

```diff
@@ -475,15 +475,43 @@ $editor-styleButton-size: 3rem;
 	padding: 10px 20px;
 }
 
+:root {
+	--preCharacterLimitt:40ch;
```

Comment:
You should scope the variable to the component instead of having it global. Since there is only one usage I wouldn't have it as a variable at all and put it inline. By the way, there's a typo in the word `limit`.

Reply:
I'm leaving the variable as it is. We might need it globally as it should be configurable through a config. If it will be implemented differently we'll get rid of the variable completelly.